.github/workflows/test.yml

```diff
@@ -76,7 +76,7 @@ jobs:
     - name: Lint tinygrad with pylint
       run: python -m pylint tinygrad/
     - name: Run mypy
-      run: python -m mypy
+      run: python -m mypy --strict-equality
```

Comment:
`--strict-equality` catches always false `==` checks like that uops err:
https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-strict-equality